android/external/MakefileTemplate

```diff
@@ -257,13 +257,27 @@ nettle-install: $(LOCAL)/lib/libnettle.so
 # BOTAN2
 
 botan.src.stamp: ../downloads/botan.src.stamp
-	cp -r ../downloads/botan botan
+	cp -r $(BOTAN_SRC) botan
 	touch $@
```

Comment:
All this stamp treatment looks weird, I commented with more detail in another comment. Also, I would never copy sources around and instead compile them from where they are. But up to you, I know, it's a lot of legacy code.

Reply:
Used botan/Makefile instead as stamp.